src/core/qgsrelation.h

```diff
@@ -272,6 +280,15 @@ class CORE_EXPORT QgsRelation
      */
     bool isValid() const;
 
+    /**
+     * Compares the two QgsRelation, ignoring the name and the ID.
+     *
+     * @param other The other relation
+     * @return true if they are similar
+     * @note added in QGIS 3.0
+     */
+    bool isSame( const QgsRelation& other ) const;
```

Comment:
`isEquivalent` sounds better to my [french] ears... maybe a native speaker has a better phrasing?